src/render/rendersettings.h

```diff
@@ -32,6 +32,8 @@ struct RenderSettings
   bool bloom = true;
   bool moreLights = true;
   bool dust = true;
+  bool dustResolutionDivisorActive = false;
```

Comment:
Maybe better combine this with dust particles in general, in the sense of `Dust Particles (<= 1/Nx =>)`. That would make the other boolean field superfluous, and having a single control for dust particles is better than two.

Reply:
Ok, i've done that and i've added 1/1 as an option so you can have the full amount of dust particles.